modules/custom/az_migration/migrations/migrate_plus.migration.az_node_event.yml

```diff
@@ -0,0 +1,190 @@
+id: az_node_event
+label: AZ Event Content
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: d7_node
+  node_type: uaqs_event
+  constants:
+    slash: '/'
+
+destination:
+  plugin: entity:node
+  bundle: az_event
+  migration_dependencies:
+    requireed:
+      - az_event_categories
+
+process:
+  nid: nid
+  vid: vid
```

Comment:
I don't think we actually want/need to preserve event node IDs or revision IDs.

Reply:
took care of this one..